### DISCUSSION TITLE
Not post scraping for account

### DISCUSSION THREAD

**Discussion by MiltonCalamaro** [kevinzg/facebook-scraper#713] [General]
2022-03-15 04:14:20, ERROR facebook_scraper.facebook_scraper Exception while requesting URL: https://m.facebook.com/bcrpoficial/posts/

**Comment by neon-ninja** [comment]
Try go to that URL in browser - it doesn't work there either. Nevertheless, this error should be logged and the scraper should fall back to extracting posts on https://m.facebook.com/bcrpoficial/. Does it not do that for you?
